docs/source/conf.py

```diff
@@ -1,72 +1,69 @@
 # Configuration file for the Sphinx documentation builder.
 #
-# This file only contains a selection of the most common options. For a full
-# list see the documentation:
+# For the full list of built-in configuration values, see the documentation:
 # https://www.sphinx-doc.org/en/master/usage/configuration.html
 
-# -- Path setup --------------------------------------------------------------
-
-# If extensions (or modules to document with autodoc) are in another directory,
-# add these directories to sys.path here. If the directory is relative to the
-# documentation root, use os.path.abspath to make it absolute, like shown here.
-
-import os
-import sys
-from pathlib import Path
-
-# sys.path.insert(0, os.path.abspath('.'))
-
-
 # -- Project information -----------------------------------------------------
+# https://www.sphinx-doc.org/en/master/usage/configuration.html#project-information
 
 project = "mamba"
-copyright = "2020, QuantStack & mamba contributors"  # noqa
+copyright = "2024, QuantStack & mamba contributors"
 author = "QuantStack & mamba contributors"
 
-# The full version, including alpha/beta/rc tags
-release = "0.5.2"
-
+version = "latest"
+release = "latest"
 
 # -- General configuration ---------------------------------------------------
+# https://www.sphinx-doc.org/en/master/usage/configuration.html#general-configuration
 
-# Load local extensions (e.g. mermaid)
-sys.path.insert(0, str(Path.cwd().resolve() / "tools"))
+extensions = []
 
-# Add any Sphinx extension module names here, as strings. They can be
-# extensions coming with Sphinx (named 'sphinx.ext.*') or your custom
-# ones.
-extensions = ["mermaid", "mermaid_inheritance", "myst_parser", "breathe"]
-
-# Configuration of Breathe Doxygen interopt
-breathe_projects = {"libmamba": os.environ.get("MAMBA_DEV_DOXYGEN_XML_DIR", "../xml")}
-breathe_default_project = "libmamba"
-breathe_default_members = ("members", "undoc-members")
-breathe_show_include = False
-
-# Add any paths that contain templates here, relative to this directory.
 templates_path = ["_templates"]
+exclude_patterns = ["_build", "Thumbs.db", ".DS_Store"]
 
-# List of patterns, relative to source directory, that match files and
-# directories to ignore when looking for source files.
-# This pattern also affects html_static_path and html_extra_path.
-exclude_patterns = []
-
+language = "en"
 
 # -- Options for HTML output -------------------------------------------------
+# https://www.sphinx-doc.org/en/master/usage/configuration.html#options-for-html-output
+
+html_theme = "alabaster"
+html_static_path = ["_static"]
+
+# The file above was generated using sphinx 8.1.3 with this command:
+# sphinx-quickstart --project "mamba" --author "QuantStack & mamba contributors" -v "latest" -r "latest" -l en --no-sep --no-makefile --no-batchfile
+# These are custom options for this project
```

Comment:
This allows to have really simple configuration file for understanding, where you can easily see, which part of the docs is the standard config, and which is custom for this project.

Updating to new sphinx versions becomes easier and overall we have better distinction between generated code and handwritten.